# Sydney Precision Data Science

This is the public repository of the Precision Bioinformatics Cluster of the Sydney Precision Data Science Centre at the University of Sydney. Here, you can access R packages developed by members of Sydney Precision Data Science covering a broad range of topics ranging from generating predictive biomarkers to single cell data analysis. 

Additionally, open analyses and data from published papers, workshops, workflows, and useful scripts are released here.

# Our Packages

|                                                                                                                                         | Name                                                                                                                            | Description                                                                                                                                                                                                                         | Single Cell        | Precision Medicine | Stats Models       |
| --------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------ | ------------------ | ------------------ |
| <img src="https://github.com/SydneyBioX/SydneyBioXStickers/blob/main/CiteFuse/CiteFuse.png?raw=true" width="100">                       | [CiteFuse](https://github.com/SydneyBioX/CiteFuse) ([paper](https://doi.org/10.1093/bioinformatics/btaa282))                 | A suite of tools for pre-processing, modality integration, clustering, differential RNA and ADT expression analysis, ADT evaluation, ligand-receptor interaction analysis, and interactive web-based visualization of CITE-seq data | :heavy_check_mark: |                    | :heavy_check_mark: |
| <img src="https://github.com/SydneyBioX/SydneyBioXStickers/blob/main/ClassifyR/classifyR.png?raw=true" width="100">                     | [ClassifyR](https://github.com/SydneyBioX/ClassifyR) ([paper](https://doi.org/10.1093/bioinformatics/btv066))                | A framework for cross-validated classification problems, with applications to differential variability and differential distribution testing                                                                                        |                    |                    | :heavy_check_mark: |
| <img src="https://github.com/SydneyBioX/SydneyBioXStickers/blob/main/CPOP/cpop_hex.png?raw=true" width="100">                           | [CPOP](https://github.com/SydneyBioX/CPOP) ([paper](https://doi.org/10.1038/s41746-022-00618-5))                             | A statistical machine learning framework for wider implementation of precision medicine                                                                                                                                             |                    | :heavy_check_mark: | :heavy_check_mark: |
|                                                                                                                                         | [DCARS](https://github.com/shazanfar/DCARS) ([paper](https://doi.org/10.1093/bioinformatics/bty698))                         | Differential correlation across ranked samples                                                                                                                                                                                      |                    |                    | :heavy_check_mark: |
|                                                                                                                                         | [directPA](https://github.com/PYangLab/directPA) ([paper](https://doi.org/10.1093%2Fbioinformatics%2Fbtt616))                | Pathway analysis in experiments with multiple perturbation designs                                                                                                                                                                  |                    |                    | :heavy_check_mark: |
| <img src="https://github.com/SydneyBioX/SydneyBioXStickers/blob/main/FuseSOM/fuseSOM_ai_standard.png?raw=true" width="100">             | [FuseSOM](https://github.com/ecool50/FuseSOM) ([paper](https://doi.org/10.1101/2023.01.18.524659))                           | A correlation based Multiview Self Organizing Map for the characterization of cell types in highly multiplexed in situ imaging cytometry assays                                                                                     | :heavy_check_mark: |                    | :heavy_check_mark: |
| <img src="https://github.com/SydneyBioX/SydneyBioXStickers/blob/main/hRUV/hRUV.png?raw=true" width="100">                               | [hRUV](https://github.com/SydneyBioX/hRUV) ([paper](https://doi.org/10.1038/s41467-021-25210-5))                             | Normalisation of multiple batches of metabolomics data in a hierarchical strategy with use of samples replicates in large-scale studies                                                                                             |                    | :heavy_check_mark: | :heavy_check_mark: |
| <img src="https://github.com/SydneyBioX/SydneyBioXStickers/blob/main/lisaClust/lisaClust_sticker_ai_standard.png?raw=true" width="100"> | [lisaClust](https://github.com/ellispatrick/lisaClust) ([paper](https://doi.org/10.1002/cyto.a.24729))                       | Clustering of local indicators of spatial association                                                                                                                                                                               | :heavy_check_mark: |                    | :heavy_check_mark: |
|                                                                                                                                         | [MoleculeExperiment](https://github.com/SydneyBioX/MoleculeExperiment) ([paper](https://doi.org/10.1186/s40168-023-01475-4)) | Provide functionality for the representation and summarisation of imaging-based spatial transcriptomics data                                                                                                                        | :heavy_check_mark: |                    |                    |
|                                                                                                                                         | [NEMoE](https://github.com/SydneyBioX/NEMoE) ([paper](https://doi.org/10.1186/s40168-023-01475-4))                           | A nutrition-aware regularized mixture of experts model                                                                                                                                                                              |                    | :heavy_check_mark: | :heavy_check_mark: |
| <img src="https://github.com/SydneyBioX/SydneyBioXStickers/blob/main/scClassify/scClassify.png?raw=true" width="100">                   | [scClassify](https://github.com/SydneyBioX/scClassify) ([paper](https://doi.org/10.15252/msb.20199389))                     | Single cell classification via cell-type hierarchies based on ensemble learning and sample size estimation.                                                                                                                         | :heavy_check_mark: |                    | :heavy_check_mark: |
| <img src="https://github.com/SydneyBioX/SydneyBioXStickers/blob/main/scDC/scDC.png?raw=true" width="100">                               | [scDC](https://github.com/SydneyBioX/scDC) ([paper](https://doi.org/10.1186/s12859-019-3211-9))                             | Perform differential composition analysis on scRNA-seq data                                                                                                                                                                         | :heavy_check_mark: |                    | :heavy_check_mark: |
| <img src="https://github.com/SydneyBioX/SydneyBioXStickers/blob/main/scdney/scdney.png?raw=true" width="100">                           | [scdney](https://github.com/SydneyBioX/scdney) ([paper](https://doi.org/10.12688/f1000research.130623.1))                   | A collection of single cell analysis R packages                                                                                                                                                                                     | :heavy_check_mark: |                    |                    |
| <img src="https://github.com/SydneyBioX/SydneyBioXStickers/blob/main/scFeatures/sticker_version2.png?raw=true" width="100">             | [scFeatures](https://github.com/SydneyBioX/scFeatures) ([paper](https://doi.org/10.1093/bioinformatics/btac590))            | Generates multi-view representations of single-cell and spatial data through the construction of a total of 17 feature types                                                                                                        | :heavy_check_mark: |                    | :heavy_check_mark: |
| <img src="https://github.com/SydneyBioX/SydneyBioXStickers/blob/main/scHOT/scHOT_hex.png?raw=true" width="100">                         | [scHOT](https://github.com/shazanfar/scHOT) ([paper](https://doi.org/10.1038/s41592-020-0885-x))                            | Single cell higher order yesting                                                                                                                                                                                                    | :heavy_check_mark: |                    | :heavy_check_mark: |
| <img src="https://github.com/SydneyBioX/SydneyBioXStickers/blob/main/scMerge/scMerge.png?raw=true" width="100">                         | [scMerge](https://github.com/SydneyBioX/scMerge) ([paper](https://doi.org/10.1073/pnas.1820006116))                         | Statistical approach for removing unwanted variation from multiple single-cell datasets                                                                                                                                             | :heavy_check_mark: |                    |                    |
| <img src="https://github.com/SydneyBioX/SydneyBioXStickers/blob/main/scReClassify/scReClassify.png?raw=true" width="100">               | [scReClassify](https://github.com/SydneyBioX/scReClassify) ([paper](https://doi.org/10.1186/s12864-019-6305-x))             | Post hoc cell type classification of single-cell RNA-sequencing data.                                                                                                                                                               | :heavy_check_mark: |                    | :heavy_check_mark: |
| <img src="https://github.com/SydneyBioX/SydneyBioXStickers/blob/main/SimBench/simbench_sticker.png?raw=true" width="100">               | [SimBench](https://github.com/SydneyBioX/SimBench) ([paper](https://doi.org/10.1038/s41467-021-27130-w))                     | Benchmark simulation methods based on two key aspects of accuracy of data properties estimation and ability to retain biological signals                                                                                            |                    |                    | :heavy_check_mark: |
| <img src="https://github.com/SydneyBioX/SydneyBioXStickers/blob/main/spicyR/spicyR_standard.png?raw=true" width="100">                  | [spicyR](https://github.com/ellispatrick/spicyR) ([paper](http://dx.doi.org/10.1186/s13059-021-02526-5))                     | Spatial analysis of in situ cytometry data.                                                                                                                                                                                         | :heavy_check_mark: |                    | :heavy_check_mark: |
| <img src="https://github.com/SydneyBioX/SydneyBioXStickers/blob/main/treekoR/treekoR.png?raw=true" width="100">                         | [treekoR](https://github.com/adam2o1o/treekoR/) ([paper](http://dx.doi.org/10.1186/s13059-021-02526-5))                      | Utilise the hierarchical nature of single cell cytometry data, to find robust and interpretable associations between cell subsets and patient clinical end points                                                                   | :heavy_check_mark: |                    | :heavy_check_mark: |
