## Repositories

### R Packages
- [CiteFuse](https://github.com/SydneyBioX/CiteFuse): A streamlined package consisting of a suite of tools for pre-processing, modality integration, clustering, differential RNA and ADT expression analysis, ADT evaluation, ligand-receptor interaction analysis, and interactive web-based visualization of CITE-seq data.
- [ClassifyR](https://github.com/SydneyBioX/ClassifyR): A framework for cross-validated classification problems, with applications to differential variability and differential distribution testing.
- [CPOP](https://github.com/SydneyBioX/CPOP): A statistical machine learning framework for transferable models in precision medicine.
- [hRUV](https://github.com/SydneyBioX/hRUV): A package for normalisation of multiple batches of metabolomics data in a hierarchical strategy with use of samples replicates in large-scale studies.
- [NEMoE](https://github.com/SydneyBioX/NEMoE): A package implementing a nutrition-aware regularized mixture of experts model addressing diet cohort heterogeneity of gut microbiome in certain diseases.

### Open Data Analyses
- [CiteFuse2020Kim](https://github.com/SydneyBioX/CiteFuse2020Kim): Scripts which reproduce all figures in our [CiteFuse manuscript](https://www.biorxiv.org/content/10.1101/854299v1).
- [NEMoE_MS](https://github.com/SydneyBioX/NEMoE_MS): Scripts which reproduce the results from the [NEMoE paper](https://www.medrxiv.org/content/10.1101/2021.11.10.21266194v1).  


### Datasets
- [BioHEART_metabolomics](https://github.com/SydneyBioX/BioHEART_metabolomics): Open heart metabolomics data.
    - Need more information about this.

### Shiny Apps and Dashboards
- [LC-N2G](https://github.com/SydneyBioX/LCN2G): An app exploring the relationship between nutrition and its corresponding gene expression data

### Workshops
- [BiocAsia2020CiteFuse](https://github.com/SydneyBioX/BiocAsia2020CiteFuse): A 2020 workshop covering multi-modal profiling of different biomolecule species in individual cells using CiteFuse.
- [BIS2019_IntroUnix](https://github.com/SydneyBioX/BIS2019_IntroUnix): A 2019 workshop which guides users through the basics of Linux.
- [BIS2019_PhosR](https://github.com/SydneyBioX/BIS2019_PhosR): A 2019 workshop illustrating some key components in PhosR package.
- [BIS2019_SC](https://github.com/SydneyBioX/BIS2019_SC): A 2019 workshop on single cell RNA-seq analysis.
- [cornell_sc_workshop](https://github.com/SydneyBioX/cornell_sc_workshop): A 2020 workshop on single cell analysis workshop at Cornell University.
- [EMBLworkshop2018](https://github.com/SydneyBioX/EMBLworkshop2018): A 2018 workshop exploring a tissue expression dataset with the hierarchical clustering and k-means clustering algorithm and visualise the dataset using heatmaps and dimension reduction tools.
- [localWorkshop](https://github.com/SydneyBioX/localWorkshop): A 2018 workshop slides and website for cross-validated classification

### Workflows
- [COVID_CCI_analysis](https://github.com/SydneyBioX/COVID_CCI_analysis): A generalizable and scalable workflow for cell type annotation, data integration and differential cell cell interaction across COVID-19 patients with distinct disease outcomes.

### Misc.
- [CPOP_public](https://github.com/SydneyBioX/CPOP_public): Assorted r-markdown scripts.
    - Need more information.
- [KidneyAllocationProcess](https://github.com/SydneyBioX/KidneyAllocationProcess): Assorted r-markdown scripts.
    - Need more information.
- [microbiome-PD](https://github.com/SydneyBioX/microbiome-PD): Microbiome Parkinson study. Assorted r-markdown scripts.
    - Need more information.